tests/recipes/halide.rs

```diff
@@ -24,23 +21,19 @@ pub fn halide_rules() -> Ruleset<Pred> {
         all_rules.clone(),
     );
     all_rules.extend(bool_only);
-
-    // Rational rules up to size 5:
     let rat_only = recursive_rules(
         Metric::Atoms,
         5,
         Lang::new(
             &["-1", "0", "1"],
             &["a", "b", "c"],
             &["-"],
-            &["+", "-", "*", "/", "min", "max"],
+            &["+", "-", "*", "min", "max"],
```

Comment:
no more div? why? if it just blows things up too much then maybe we should have it here but not in the `all` workload? it's hard to imagine that learning simple arithmetic rules with div could be *worse*?

Reply:
it's not worse, but it gets you nothing and adds like 300 rules

Reply:
difference between 910 and 1232 rules. I feel like 910 looks better if proving power is the same

Reply:
whoa that's a big difference. ok leave it out.